core/controllers/base.py

```diff
@@ -383,16 +382,15 @@ def render_template(self, filepath, iframe_restriction='DENY'):
         # deploy a new version, using only 'no-cache' doesn't work properly.
         self.response.cache_control.no_store = True
         self.response.cache_control.must_revalidate = True
-        self.response.headers[b'Strict-Transport-Security'] = (
-            b'max-age=31536000; includeSubDomains')
-        self.response.headers[b'X-Content-Type-Options'] = b'nosniff'
-        self.response.headers[b'X-Xss-Protection'] = b'1; mode=block'
+        self.response.headers['Strict-Transport-Security'] = (
+            'max-age=31536000; includeSubDomains')
+        self.response.headers['X-Content-Type-Options'] = 'nosniff'
+        self.response.headers['X-Xss-Protection'] = '1; mode=block'
 
         if iframe_restriction is not None:
             if iframe_restriction in ['SAMEORIGIN', 'DENY']:
                 self.response.headers[
-                    b'X-Frame-Options'] = python_utils.convert_to_bytes(
-                        iframe_restriction)
+                    'X-Frame-Options'] = str(iframe_restriction)
```

Comment:
TBH, I think this change is fine since next step after the migration is to get rid of `convert_to_bytes`. And here `str` is required instead of `bytes` because of `webapp2`.